xbmc/input/mouse/MouseStat.h

```diff
@@ -53,6 +53,14 @@ enum MOUSE_BUTTON
   MOUSE_EXTRA_BUTTON4
 };
 
+enum class HoldAction
+{
+  NONE = 0, //no action should occur
```

Comment:
Can you remove the `= 0`? We always static_cast, so the interpretation of the enum as an int can happen there.

Reply:
done